src/util/std_expr.h

```diff
@@ -1401,8 +1325,7 @@ inline void validate_expr(const equal_exprt &value)
 }
 
 
-/*! \brief inequality
-*/
+/// \brief inequality
```

Comment:
To be strict, this is 'disequality'

Reply:
https://en.wikipedia.org/wiki/Inequality_(mathematics)

Reply:
Changed to `disequality`, as it seems important to @peterschrammel ;-)